src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -292,7 +292,12 @@ final class SecureStore: Store {
 		get { kvStore["traceLocationsInfoScreenShown"] as Bool? ?? false }
 		set { kvStore["traceLocationsInfoScreenShown"] = newValue }
 	}
-
+	
```

Comment:
perhaps we should introduce here a separate extension with a new protocol for all the event stuff and move this and the property above in it?

Reply:
I noticed we have the same for 
```
	var checkinInfoScreenShown: Bool {
		get { kvStore["checkinInfoScreenShown"] as Bool? ?? false }
		set { kvStore["checkinInfoScreenShown"] = newValue }
	}
	
	var traceLocationsInfoScreenShown: Bool {
		get { kvStore["traceLocationsInfoScreenShown"] as Bool? ?? false }
		set { kvStore["traceLocationsInfoScreenShown"] = newValue }
```
	so i just followed it, also artur told me to add it to the general store so i don't know, what do you think ?

Reply:
I like your approach, but not the name. Because if we gather all event registration stuff in that extension, "caching" would be for me misleading.
So perhaps only "EventRegistration"?